src/main/java/seedu/address/logic/commands/DocumentContentAddCommand.java

```diff
@@ -31,7 +31,7 @@
             + "[" + PREFIX_NOTE_CONTENT + "Note] "
```

Comment:
Should we make this a mandatory parameter instead of optional, since the finish command cannot be executed without it?